README.md

```diff
@@ -87,6 +87,25 @@ argument.
 CMake might have a hard time finding the correct Python version because Apple. Ping me `@jeffniu22`
 if you've got any issues.
 
+# Setting up Minotaur on Ubuntu 16.04
+From a fresh install, you will need these packages
+
+```bash
+sudo apt install build-essential cmake python3 python3-dev qt5-default libudev-dev git
+```
+
+Then clone the repository and build with
+```bash
+git clone https://github.com/uwnrg/minotaur-cpp.git
+cd minotaur-cpp
+mkdir build
+cd build
+cmake ../
+make
+```
+
+And then run with `./minotaur-cpp`
```

Comment:
can this be generalized for any linux platform?

Reply:
I'm not really familiar enough with Linux, but I assume that older Ubuntu versions would need to add PPAs to install the dependencies

Reply:
It basically says "Works for sure on this version, probably just similar steps on different ones"

Reply:
I meant different linux distributions (like arch, fedora, debian, etc) not other versions of ubuntu. You may wanna make a general list of dependencies, and people can look up how to download it in their respective distributions.